.travis.yml

```diff
@@ -6,16 +6,16 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
   - hhvm
-  - nightly
 env:
   - COMPOSER_OPTS=""
   - COMPOSER_OPTS="--prefer-lowest"
 
 matrix:
   allow_failures:
+    - php: 7.3
```

Comment:
As PHP 7.3 has reached "Release Candidates" versions, I believe we can drop this. But, I'll leave to the maintainers.